lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3888,25 @@ HTML
 						;
 				}
 
+				if ($packager_codes{$canon_tagid}{cc} eq 'fr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{raison_sociale_enseigne_commerciale}<br>
+$packager_codes{$canon_tagid}{adresse} $packager_codes{$canon_tagid}{code_postal} $packager_codes{$canon_tagid}{commune}<br>
+SIRET : $packager_codes{$canon_tagid}{siret} - <a href="$packager_codes{$canon_tagid}{section}">Source</a>
+</p>
+HTML
+						;
+				}
+
+				if ($packager_codes{$canon_tagid}{cc} eq 'hr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{naziv odobrenog objekta    (approved establishment)}<br>
+$packager_codes{$canon_tagid}{adresa objekta (address)} $packager_codes{$canon_tagid}{mjesto i poštanski broj (town/postal code)} ($packager_codes{$canon_tagid}{županija (county)})
```

Comment:
`Bareword "town" not allowed while "strict subs"`
might come from this:

```suggestion
$packager_codes{$canon_tagid}{'adresa objekta (address)'} $packager_codes{$canon_tagid}{'mjesto i poštanski broj (town/postal code)'} ($packager_codes{$canon_tagid}{'županija (county)'})
```

Reply:
Thanks @alexgarel!

I updated the header to remove spaces, parenthesis, dots, etc.